packages/@o3r/azure-tools/.compodocrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://raw.githubusercontent.com/compodoc/compodoc/develop/src/config/schema.json",
+  "name": "Azure tools",
+  "output": "../../../generated-doc/azure-tools",
+  "tsconfig": "./tsconfig.doc.json",
+  "assetsFolder": "../../../.attachments",
```

Comment:
we need to keep in mind that these needs to be changed when `main` will be cascaded to v10